vizro-core/src/vizro/models/_navigation/accordion.py

```diff
@@ -44,7 +42,7 @@ def build(self, *, active_page_id=None):
         # Note build does not return _NavBuildType but just a single html.Div with id="nav-panel".
         # Hide navigation panel if there is only one page
         if len(list(itertools.chain(*self.pages.values()))) == 1:
-            return html.Div(hidden=True, id="nav-panel")
+            return dbc.Nav(id="nav-panel", className="d-none invisible")
```

Comment:
`d-none` is equivalent to `display: none`
`invisible` is equivalent to `visibility: hidden`

https://getbootstrap.com/docs/5.0/utilities/display/